frontend/public/style/_common.scss

```diff
@@ -457,6 +457,10 @@ $co-external-link-padding-right: 15px;
   padding: 0 0 30px 0;
 }
 
+.co-hide-no-active-namespace > tbody > tr > :nth-child(2) {
```

Comment:
None of the cluster-scoped resources have a namepsace column, so this would randomly hide the 2nd column in a lot of tables.

I think a more robust approach is to something like

```css
.co-no-active-namespace .co-hide-no-active-namespace {
  display: none;
}
```

And then just use `co-hide-no-active-namespace` in the list pages for both table data and headers. It requires updating each list page, but it's a small change and safer.

I think we'll need to update all the list pages anyway because many still use Bootstrap grid. For those, hiding the namespace column doesn't help. They'll just be unused space to the right of the table.

One awkward thing is that at mobile we'll only have the name column. It would be nice to have an option to show something else, at least for important resources (like pod status for pods).